### DISCUSSION TITLE
Facing Issue When i Run Npm Run Server With SunbirdEd-portal  after completions all procedure

### DISCUSSION THREAD

**Discussion by om-666** [Sunbird-Ed/Community#568] [Bugs] [answered]
I am encountering an issue when I run "npm run server" with SunbirdEd-portal after completing all the procedures. And my nodemon is crashed 

I believe this error is occurring due to a JavaScript heap out of memory issue. I think some changes need to be made in the Package.json file. If anyone has a solution to this issue, please share it.
 
 
![issue2](https://github.com/Sunbird-Ed/Community/assets/104663400/8ccbebf6-e41d-479a-907f-95f610ee2f27)
![Issue](https://github.com/Sunbird-Ed/Community/assets/104663400/7bb94bd0-93ee-42f6-a190-ed0ae6bc5d90)

**Comment by Krishnaj20** [comment, marked as answer]
@chitranshu-keshav @gandham-santhosh - Please check and respond to this query. It has already been 2 days without any response.

@vinukumar-vs @rajeevsathish

**Reply by rajeevsathish** [reply]
@om-666 your RAM is not enough to run the portal in local that is why the heap error is coming increase the RAM for js and then try

**Reply by om-666** [reply]
issue resolved Thank u Rajeev
